include/zephyr/bluetooth/conn.h

```diff
@@ -1329,6 +1451,67 @@ int bt_conn_le_subrate_set_defaults(const struct bt_conn_le_subrate_param *param
 int bt_conn_le_subrate_request(struct bt_conn *conn,
 			       const struct bt_conn_le_subrate_param *params);
 
+/** @brief Read Minimum Supported Connection Interval Groups.
+ *
+ *  Read the minimum supported connection interval and supported interval
+ *  groups from the local controller. This information describes what the
+ *  local controller supports.
+ *
+ *  @note To use this API @kconfig{CONFIG_BT_SHORTER_CONNECTION_INTERVALS} must be set.
+ *
+ *  @param info Pointer to structure to receive the minimum connection interval
+ *              information.
+ *
+ *  @return Zero on success or (negative) error code on failure.
+ */
+int bt_conn_le_read_min_conn_interval_groups(struct bt_conn_le_min_conn_interval_info *info);
```

Comment:
I think "action" should be at the end of the function call, <bt_conn_le_read_min_conn_interval_groups> should be  <bt_conn_le_min_conn_interval_groups_read> then.